tests/Homework 16/registration.spec.js

```diff
@@ -0,0 +1,286 @@
+            return window.getComputedStyle(element).borderColor;
+        })
+        await expect(borderColor).toBe('rgb(220, 53, 69)')
+        await email.clear()
+
+
+    });
+    test('Password validation', async () => {
+
+        const loginBtn = page.locator('button', {
+            hasText: 'Sign in'
+        })
+        const registrationBtn = page.locator('button', {
+            hasText: 'Registration'
+        })
+
+        await loginBtn.click();
+        await registrationBtn.click();
+
+        const password = page.locator('#signupPassword')
+        const error = page.locator('div.invalid-feedback')
+
+        await password.fill('')
+        await password.blur()
+        await expect(error).toHaveText('Password required')
+        await password.clear()
+
+
+        await password.fill('111')
+        await password.blur()
+        await expect(error).toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter')
+        await password.clear()
+
+
+        await password.fill('111')
+        await password.blur()
+        await page.waitForTimeout(100);
+        const borderColor1 = await password.evaluate((element) => {
+            return window.getComputedStyle(element).borderColor;
+        })
+        await expect(borderColor1).toBe('rgb(220, 53, 69)')
+        await password.clear()
+
+
+
+    });
+    test('Password confirmation validation', async () => {
+
+        const loginBtn = page.locator('button', {
+            hasText: 'Sign in'
+        })
+        const registrationBtn = page.locator('button', {
+            hasText: 'Registration'
+        })
+
+        await loginBtn.click();
+        await registrationBtn.click();
+
+        const password = page.locator('#signupPassword')
+        const passwordConfirmation = page.locator('#signupRepeatPassword')
+        const error = page.locator('div.invalid-feedback')
+
+
+        await passwordConfirmation.fill('')
+        await passwordConfirmation.blur()
+        await expect(error).toHaveText('Re-enter password required')
+        await passwordConfirmation.clear()
+
+
+        await passwordConfirmation.fill('111')
+        await passwordConfirmation.blur()
+        await expect(error).toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter')
+        await passwordConfirmation.clear()
+
+
+        await password.fill('1321@Helena13')
+        await passwordConfirmation.fill('1321@Helena')
+        await passwordConfirmation.blur()
+        await expect(error).toHaveText('Passwords do not match')
+        await passwordConfirmation.clear()
+        await password.clear()
+
+
+        await passwordConfirmation.fill('111')
+        await passwordConfirmation.blur()
+        await page.waitForTimeout(100);
+        const borderColor2 = await passwordConfirmation.evaluate((element) => {
```

Comment:
[toHaveCSS](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-css)